bandg/65309.js

```diff
@@ -0,0 +1,18 @@
+module.exports = [
+  {
+    source: 'Battery Status',
+    node: function(n2k){
+      return `instruments.wireless.${n2k.src}.batteryStatus`
```

Comment:
I don't think using `src` as the instance identifier makes sense, as it can change essentially randomly when the bus reshuffles addresses. @sbender9 should this be `canId`?

Reply:
Yeah, it should probably use canId. The canId may not be there initially, so should ignore messages until it is there.